ecosystem/typescript/sdk/src/api/data-contracts.ts

```diff
@@ -234,17 +234,17 @@ Generic type parameter value example, the following is `0x1::TransactionFee::Tra
             {"constraints": [], "is_phantom": true}
         ],
         "fields": [
-            { "name": "balance", "type": "0x1::Aptos::Aptos<T0>" },
-            { "name": "preburn", "type": "0x1::Aptos::Preburn<T0>" }
+            { "name": "balance", "type": "0x1::Aptos::Aptos<T0 />" },
```

Comment:
Maybe your IDE is misconfigured. This file should never been updated. It is an generated file. The extra "/" doesn't make sense. These "<" and ">" are not html/xml tags.

Reply:
I know. I am running into two problems and I am trying to root cause them. 

1. First on my local, I get a `Expected corresponding JSX closing tag` error, which I think is because Docusaurus generates incorrect Markdown. Just to test this theory, I manually added the extra space in the source and the error went away. Docs generate fine.  See this, and a few others like this: https://bobbyhadz.com/blog/react-expected-corresponding-jsx-closing-tag 
2. Second, when I push to the repo, the Netlify deploy preview fails with `Cannot find module 'axios' or its corresponding type declarations.` error. These dependencies are properly present in yarn.lock and package.json but something in the Docusaurus plugin for this API is messing it up. 
3. If I can't solve this issue, then we have to forget about Docusaurus API docs and switch to the standard Typedoc docs that you originally used. In any case, for now I will pause this debugging, I need to focus on my AIT-2 docs.